api-spec/protobuf/operator.proto

```diff
@@ -236,6 +256,13 @@ enum TradeStatus {
   EXPIRED = 5;
 }
 
+enum ActionType {
+  TRADE_SETTLED = 0;
+  ACCOUNT_LOW_BALANCE = 1;
```

Comment:
we want to distringuish between fee account and normal market account?

Reply:
I thought it wasn't necessary... ideally, the payload of such request would include the account index to identify which account has low balance, along with other info like the account balance and, in case of a market, the asset pair.

In the past, it was the blockchain listener to signal through a log on the stdout when the fee/market's balance was too low. Now we need to put in place a proper logic; at the moment the only check the daemon performs on balances should be on TradePropose/MarketPrice, by checking if the balance is <= 0. This is the reason why you won't find the webhook invocation for the event AccountLowBalance.